packages/react/src/components/Tag/Tag.js

```diff
@@ -36,13 +44,14 @@ const Tag = ({ children, className, type, filter, disabled, ...other }) => {
     <span
       className={tagClasses}
       title="Clear filter"
+      role={listItem && 'listitem'}
```

Comment:
```suggestion
      role={listItem ? 'listitem' : undefined}
```

Would it need to be this?